clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -55,7 +55,10 @@ async function initialize(context: vscode.ExtensionContext) {
   try {
     await vscode.workspace.fs.createDirectory(context.globalStorageUri);
   } catch (_error) {
```

Comment:
The convention to start a variable with an underscore like in `_error` is for letting the tooling know it is unused and it is intentional (I am sure you know).

But now we actually use the variable so the underscore should be removed.

Reply:
should resolve : 83f21931e1b227c9067719b2eb37d9de80e20cfe